packages/components/src/utils/isVenueHelsinkiCityOwned.ts

```diff
@@ -0,0 +1,24 @@
+import { ProviderType, ServiceOwnerType } from '../types';
+import type { UnifiedSearchVenue, Venue } from '../types';
+
+/**
+ * Check is venue owned by City of Helsinki
+ * @param venue
+ * @return {boolean}
+ */
+export default function isVenueHelsinkiCityOwned(
+  venue: UnifiedSearchVenue | Venue
+): boolean {
+  if (venue.__typename == 'UnifiedSearchVenue') {
+    return (
+      venue.serviceOwner?.providerType == ProviderType.SelfProduced &&
+      venue.serviceOwner?.type == ServiceOwnerType.MunicipalService
+    );
```

Comment:
also this could have a type check function like described earlier for events and venues.

Reply:
Taken into account in ea2b90da2f5c23a2b3a06d4586b31e6e964c9caa